spec/user_interface_spec.rb

```diff
@@ -29,25 +37,37 @@
     end
   end
 
+  describe 'assign_color' do
+    it 'sets X to magenta and O to green' do
+      expect(user_interface.instance_variable_get(:@colors)).to eql({"X"=>:magenta, "O"=>:green})
```

Comment:
Is there a way to test with without using `instance_variable_get`?

Reply:
I wondered if I should even test that.  I added a bit to some [tests](https://github.com/saralein/tic-tac-toe_ruby/pull/18/files#diff-7aa5a624018bd299e043383e4af27151R162) to indirectly see if it's working by checking that colors were called when X or O wins.